doc/readme.md

```diff
@@ -0,0 +1,335 @@
+
+![Next-Level](figs/next-level.gif)
+
+Schauen wir einmal in die Methode `Game#frame`, wird dort die Methode `loadNextLevelIfEntityIsOnEndTile` aufgerufen.
+
+```java
+    /** Called at the beginning of each frame. Before the controllers call <code>update</code>. */
+    protected void frame() {
+        setCameraFocus();
+        manageEntitiesSets();
+        getHero().ifPresent(this::loadNextLevelIfEntityIsOnEndTile);
+        if (Gdx.input.isKeyJustPressed(Input.Keys.P)) togglePause();
+    }
+```
+
+Diese Methode prüft, ob sich der Held auf der Leiter befindet. Wenn ja, wird über die `LevelAPI` ein neues Level geladen.
+
+```java
+    private void loadNextLevelIfEntityIsOnEndTile(Entity hero) {
+        if (isOnEndTile(hero)) levelAPI.loadLevel(LEVELSIZE);
+    }
+```
+
+Beim Laden eines neuen Levels wird automatisch die `Game#onLevelLoad`-Methode aufgerufen.
+
+```java
+    @Override
+    public void onLevelLoad() {
+        currentLevel = levelAPI.getCurrentLevel();
+        entities.clear();
+        getHero().ifPresent(this::placeOnLevelStart);
+    }
+```
+
+In der Standardimplementierung werden dabei alle Entitäten entfernt und wenn es einen Helden gibt, wird dieser auf die Startposition des neuen Levels platziert und wieder in das Spiel geladen.
+
+### Existierenden Helden analysieren
+
+Wenn Sie wollen, können Sie mit Ihrem Helden weiterarbeiten, oder Sie holen sich den ebereits existierenden Helden zurück.
+
+Der vorgwfertigte Held hat zwei weitere Besonderheiten, die Sie noch nicht kennen. Zum einem besitzt er einen Feuerball-Skill. Dieser wird in `Hero#setupFireballSkill` erstellt.
+
+```java
+    private void setupFireballSkill() {
+        firstSkill =
+                new Skill(
+                        new FireballSkill(SkillTools::getCursorPositionAsPoint), fireballCoolDown);
+    }
+```
+
+In der Standardkonfiguration können SIe den Feuerball in richung ihres Mauscursors mit der Taste R abfeuer. Mehr zum Thema Skill finden Sie [hier](ecs/skills/readme.md).
+
+Zusätzlich besitzt der Held ein `HitboxComponent`. Damit kann er mit anderen Entitäten, die ebenfalls ein `HitboxComponent` haben, kolidieren.
+
+```java
+    private void setupHitboxComponent() {
+        new HitboxComponent(
+                this,
+                (you, other, direction) -> System.out.println("heroCollisionEnter"),
+                (you, other, direction) -> System.out.println("heroCollisionLeave"));
+    }
+```
+
+Wenn Sie mehr dazu erfahren wollen, schauen Sie [hier](ecs/components/hitbox_component.md).
+           
+## Linksammlung
+
+Hier finden Sie noch einige Links um Ihr Verständnis des Dungeons zu vertiefen:
+
+- [ECS Basics](ecs/ecs_basics.md)
+- [Eigener Inhalt](ecs/create_own_content.md)
+- [Übersicht der Systeme](ecs/systems/readme.md)
+- [Übersicht der Components](ecs/components/readme.md)
+- [Übersicht der Spielelemente](ecs/gameelements/)
```

Comment:
Der Unterordner "gameelements/" in "ecs/" existiert nicht. 

Bitte am besten immer auf ein "readme.md" und nicht einen Ordner verlinken. Technisch sollte GitHub zwar damit klarkommen, eine Reihe von Tools aber scheinbar nicht - mein IntelliJ beschwert sich beispielsweise :)

Reply:
Hmmm. Der Unterordner wird nur hier im PR nicht angezeigt, in der Konsole ist er da. Frage mich, warum IJ ein Problem hat: Ist da irgendwas falsch geschrieben? Oder ist es doch die Sache mit Order vs. Datei im Link?